Server/JokesOnYou.Web.Api/Services/Interfaces/IUserService.cs

```diff
@@ -9,9 +9,11 @@ namespace JokesOnYou.Web.Api.Services.Interfaces
 {
     public interface IUserService
     {
-        Task<IEnumerable<User>> GetAll();
-        Task<User> GetUserById(string id);
-        Task UpdateUser(UserUpdateDTO updateDTO);
         Task DeleteUser(string id);
+        Task<User> CreateUser(UserRegisterDTO registerDTO);
+        Task<UserReplyDTO> GetUserReplyById(string id);
+        Task UpdateUser(User user);
```

Comment:
The UserService will accept an UpdateDto ( I should really change all DTO's to Dto) and then map it to the User;